### ISSUE TITLE
[ADE Support]: VA Notify : VA Notify Portal: 508 Audit Report Findings - Defect 11 | Critical

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#111678]
> [!NOTE]
> **QUICK SUMMARY:** _This ticket is the child ticket for https://github.com/department-of-veterans-affairs/va.gov-team/issues/110877 . The ticket contains information on all of the defects found by the 508 office in relation to WCAG 1.3.1. Some issues were resolved during the first round of feedback, and they have been crossed out below. The team is looking for guidance on how to resolve the remaining issues (4 in total). Links to the staging site where examples of the issues can be found are listed in the ticket._

### Defect 11 | Critical
_WCAG 4.1.2 Name, Role, Value | For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies._

When the Your session expired. Please sign in again to continue. message appears, no role is read by assistive technology for the Close Message element—“Close Message Graphic” is read. Similarly, no role is read for Close Message element after creating a template (Accessibility Testing Service 1 -> Add Template -> Email -> Create template). (New: 05/28/2025)

[https://staging.notifications.va.gov/](https://urldefense.com/v3/__https://staging.notifications.va.gov/__;!!K2n62VMvFw!FxwZ1EPdR6LXeJQPKjd4spootk8jry8DAd6PSt0Q8seiOX4UJEEvalQGKqPlBp5NhHbpWsEtj2Wj-P-6TbI-oqc4czg$) (After session expire time). There are multiple banners throughout the portal

On the Untitled Notification | VA Notify screen (Accessibility Testing Service 1 -> Add Template -> Email), no role is read by assistive technology when tab is used to move the focus to Please make sure your notification follows the Email Content and Style Guide. (New: 05/28/2025)

[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content__;!!K2n62VMvFw!FxwZ1EPdR6LXeJQPKjd4spootk8jry8DAd6PSt0Q8seiOX4UJEEvalQGKqPlBp5NhHbpWsEtj2Wj-P-6TbI-AMOPoto$)

On the Accessibility Testing Service 1 Templates | VA Notify screen (Accessibility Testing Service 1 -> Delete Template(s)), the state of the check boxes cannot be understood by assistive technology. For example, when navigating the table with assistive technology table commands and selecting the fifth row, “Select SMS Template 2 check box not checked” is read for cell (5,1). Note that no names are read when using the shortcut x to navigate to a check box. (New: 05/28/2025)

[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/archive](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/archive__;!!K2n62VMvFw!FxwZ1EPdR6LXeJQPKjd4spootk8jry8DAd6PSt0Q8seiOX4UJEEvalQGKqPlBp5NhHbpWsEtj2Wj-P-6TbI-TrwF5cU$)

### Ticket Updates
- [x] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [x] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._
